nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/resources/nifi/fingerprint/flow2.xml

```diff
@@ -21,6 +21,8 @@
         <position x="0.0" y="0.0"/>
         <style/>
         <comment/>
+        <flowfileConcurrency>UNBOUNDED</flowfileConcurrency>
+        <flowfileOutboundPolicy>STREAM_WHEN_AVAILABLE</flowfileOutboundPolicy>
```

Comment:
Are these additional elements required for this change?

Reply:
Thanks again @exceptionfactory for the review. No, those changes are not required. During development of flowfile concurrency feature for a process group, the flow configuration files weren't updated. I thought of adding those since I'm anyway modifying those files. Just updating the files to what it looks like today in flow.xml.gz as it also gets fingerprinted. I can revert back the change if it is not relevant to this PR. Please let me know

Reply:
Thanks for the confirmation @naddym, that's helpful to know. Leaving the changes seems acceptable, I just wanted to confirm that they weren't necessary for the fingerprint calculation.

Since the primary change is correcting the code to evaluate `groupAccessControl` properly, wanted to make sure I wasn't missing something.

Reply:
Thanks @exceptionfactory. That was a valid question. I should have been more specific while describing the PR on additional changes. Will take care of it from next time. Thanks again.